gc/base/VirtualMemory.hpp

```diff
@@ -191,6 +199,7 @@ class MM_VirtualMemory : public MM_BaseVirtual {
  * friends
  */
 	friend class MM_MemoryManager;
+	friend class MM_SparseVirtualMemory;
```

Comment:
i'd rather move some members from private to protected than make the subclass a friend (or something else is an issue?)

Reply:
Ok, will do 👍  No there is nothing else that should be an issue.